testsuite/single_node_performance.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+
+# Copyright © Aptos Foundation
+# SPDX-License-Identifier: Apache-2.0
+
+import subprocess
+import re
+import os
+import tempfile
+from tabulate import tabulate
+
+EXPECTED_TPS = {
+    ("no-op", False): 18000.0,
+    ("coin-transfer", False): 12000.0,
+    ("coin-transfer", True): 17000.0,
+    ("account-generation", False): 10000.0,
+    ("account-generation", True): 15500.0,
+    ("create-new-account-resource", False): 12300.0,
+    ("modify-global-resource", False): 4000.0,
+    ("modify-ten-global-resources", False): 10700.0, 
+    ("large-module-working-set-no-op", False): 3600.0,
+    ("publish-package", False): 160.0,
+    # ("token-v1nft-mint", False): 2000.0
+}
+
+NOISE_FRACTION = 0.1
+
+# assert on production TPS
+CONCURRENCY_LEVEL = 8
+BLOCK_SIZE = 10000
+NUM_BLOCKS = 10
```

Comment:
why only 10 blocks?

Reply:
I want this for landblocking, and needs to be fast. with 10 it seems stable, might do 20.

for IOPS we need much much more, I might do that for a single transaction type (like account_creation), not for all.